x-pack/plugins/security_solution/server/lib/detection_engine/rules/bulk_actions/split_bulk_edit_actions.ts

```diff
@@ -29,6 +29,10 @@ export const splitBulkEditActions = (actions: BulkActionEditPayload[]) => {
 
   return actions.reduce((acc, action) => {
     switch (action.type) {
+      case BulkActionEditType.set_schedule:
```

Comment:
updating the schedule needs to update both `attributes` and `params`: `interval` and the `meta.from` and `from` params, respectively